service/src/main/java/greencity/service/SocialNetworkImageServiceImpl.java

```diff
@@ -57,13 +54,13 @@
      * @return {@link SocialNetworkImage}
      */
     @Override
-    public SocialNetworkImage getSocialNetworkImageByUrl(String url) {
+    public SocialNetworkImageVO getSocialNetworkImageByUrl(String url) {
         try {
             URL checkUrl = new URL(url);
-            Optional<SocialNetworkImage> optionalSocialNetworkImage =
+            Optional<SocialNetworkImageVO> optionalSocialNetworkImageVO =
                 findByHostPath(checkUrl.getHost());
-            if (optionalSocialNetworkImage.isPresent()) {
-                return optionalSocialNetworkImage.get();
+            if (optionalSocialNetworkImageVO.isPresent()) {
```

Comment:
Use ternary expression ove if/else statement. It is more readable

Reply:
done